Test/Fragments/ContactFragment.cs

```diff
@@ -0,0 +1,175 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+
+using Android.App;
+using Android.Content;
+using Android.OS;
+using Android.Runtime;
+using Android.Util;
+using Android.Views;
+using Android.Widget;
+using Android.Provider;
+using Test.Models;
+using Test.Api;
+using System.Net;
+using Android.Graphics;
+using Java.IO;
+using System.IO;
+
+namespace Test.Fragments
+{
+    public class ContactFragment : Fragment
+    {
+        private View view;
+
+        private List<CheckBox> checkBoxs;
+
+        private EditText firstName;
+
+        private EditText lastName;
+
+        private EditText email;
+
+        private EditText company;
+
+        private EditText position;
+
+        private EditText branch;
+
+        private EditText comments;
+
+        private Spinner spinner;
+
+        private string reason;
+
+        private string imageBase64;
+
+        private Contact contact = new Contact();

```

Comment:
`Contact` can be the in method variable. It's better

Reply:
Done